### PR TITLE
version from __init__.py

### PR DESCRIPTION
**Pull request by sachindavra**
Signed-off-by: sachindavra <raju.davra@gmail.com>

### CODE REVIEW

**Comment by sachindavra** [comment]
Kindly try running it and let me know in case modification is required.

File: src/svfrontend/main.py [hunk 1]
```diff
@@ -421,7 +422,7 @@ def termpage(contiden):
 @click.option("-p", "--portdata", "portdata", help="Set the port value [0-65536]", default="9696")
 @click.option("-6", "--ipprotv6", "netprotc", flag_value="ipprotv6", help="Start the server on an IPv6 address")
 @click.option("-4", "--ipprotv4", "netprotc", flag_value="ipprotv4", help="Start the server on an IPv4 address")
-@click.version_option(version="1.1.0-beta", prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
+@click.version_option(__version__="1.1.0-beta", prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
```
Comment (t0xic0der) [inline src/svfrontend/main.py:425]: There is no point in importing the `__version__` variable from the `__init__.py` file if it is being reassigned to a new value here. The version number is, hence, still hardcoded. Please make the necessary changes.

File: src/svfrontend/main.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 from hashlib import sha256
 from json import dumps
 from time import time
+from __init__ import __version__
```
Comment (t0xic0der) [inline src/svfrontend/main.py:25]: You need not import the `__version__` variable again as it has been imported already in the code via the lines from 38 to 43.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Please make the necessary changes to use the already imported version variable.

**Comment by sachindavra** [comment]
Hi @t0xic0der, thanks for guidance. 
I have made the changes and created pull request. Please check and let me know if still changes are required.

File: src/svfrontend/main.py [hunk 3]
```diff
@@ -421,7 +421,7 @@ def termpage(contiden):
 @click.option("-p", "--portdata", "portdata", help="Set the port value [0-65536]", default="9696")
 @click.option("-6", "--ipprotv6", "netprotc", flag_value="ipprotv6", help="Start the server on an IPv6 address")
 @click.option("-4", "--ipprotv4", "netprotc", flag_value="ipprotv4", help="Start the server on an IPv4 address")
-@click.version_option(version="1.1.0-beta", prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
+@click.version_option(version=__version__, prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
```
Comment (t0xic0der) [inline src/svfrontend/main.py:424]: Please change this to the following code, as the `__version__` variable has been imported from the `__init__.py` file as `frntvers`.

```suggestion
@click.version_option(version=frntvers, prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
```

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Kindly make the requested changes. :)

File: src/svfrontend/main.py (hunk 3 above)
Comment (sachindavra) [reply src/svfrontend/main.py]: Thanks for suggesting the changes. 
I have made the changes and raise a pull request. Please review it and let me know if changes are required anymore.

File: src/svfrontend/main.py [hunk 4]
```diff
@@ -421,7 +421,7 @@ def termpage(contiden):
 @click.option("-p", "--portdata", "portdata", help="Set the port value [0-65536]", default="9696")
 @click.option("-6", "--ipprotv6", "netprotc", flag_value="ipprotv6", help="Start the server on an IPv6 address")
 @click.option("-4", "--ipprotv4", "netprotc", flag_value="ipprotv4", help="Start the server on an IPv4 address")
-@click.version_option(version="1.1.0-beta", prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
+@click.version_option(version=frntvers, prog_name=click.style("SuperVisor Frontend Service", fg="magenta"))
```
Comment (t0xic0der) [inline src/svfrontend/main.py:424]: LGTM. :+1:

**Review by t0xic0der — APPROVED** [review]
I am merging this PR now.

As the commit messages were not accurately representing the changes which took place in them and just because the scope of this PR is limited, I would squash the commits into one and merge them.

Thanks for contributing. Feel free to star the repo if you found it useful.

**Pull request merged**